cmake/platforms/OSX.cmake

```diff
@@ -30,7 +30,7 @@ set(CMAKE_FIND_ROOT_PATH_MODE_INCLUDE ONLY)
 # Sysroot clang
 set(OSX_EXTRA_FLAGS
 	" -target ${TOOLCHAIN_PREFIX}"
-	" -mmacosx-version-min=11"
+	" -mmacosx-version-min=10.15"
```

Comment:
Why downgrade?

Reply:
downgrade? we are moving the min version from 10.15 to 11.0. I think  bitcoin uses 13 as the minimum version now. I'll confirm.